packages/test/test-end-to-end-tests/src/test/summarizeIncrementally.spec.ts

```diff
@@ -98,7 +98,7 @@ function validateDDSStateInSummary(
  */
 describeCompat(
 	"Incremental summaries for data store and DDS",
-	"FullCompat",
+	"2.0.0-internal.1.4.9" /** equivalent to FullCompat. Currently used for testing purposes on ADO pipelines */,
```

Comment:
This is not completely equivalent to FullCompat because we aren't going to run CrossVersion, right?